src/marshmallow/fields.py

```diff
@@ -614,6 +621,13 @@ def _nested_normalized_option(self, option_name: str) -> list[str]:
             if field.startswith(nested_field)
         ]
 
+    # this helper defers the import of `marshmallow.schema` to avoid circular imports
+    # however, it otherwise aliases Schema.from_dict
+    def _nested_from_dict(self, fields: dict[str, Field | type]) -> type:
```

Comment:
Wouldn't it work the same by just adding the `Schema` import in` __init__` and calling `Schema.from_dict` from there?

Functionally equivalent, but less awkward, perhaps. In any case, just a matter of preference, no big deal.


```py
            if isinstance(nested, dict):
                from marshmallow.schema import Schema
                nested = Schema.from_dict(nested)
```

Reply:
Yeah, that would work equivalently-well. Before I added the annotations, I thought it would be nice to isolate this into a dedicated method, but it does look a bit messy now.

I'll switch to just doing the import in the spot you suggested. I think it will read more nicely.